gnucash/gnome-utils/gnc-cell-renderer-text-flag.c

```diff
@@ -0,0 +1,222 @@
+/**
+ * gnc-cell-renderer-text-flag.h -- text cell renderer with flag.
+ *
+ * Note that this widget will track changes in the account tree, and update
+ * itself accordingly.  If an account with the same name exists in the
+ * freshly-retrieved account list, the widget will re-select that account.
```

Comment:
As far as I understand from reading the code, there is nothing in here to actually track accounts. It merely has a property "flagged" which can be set by the caller.
Please move this comment elsewhere that actually implements this.

Reply:
Sorry, it was a paste error. I copied the copyright notice from another file and left that part unchanged.